src/snowcli/cli/streamlit/commands.py

```diff
@@ -112,35 +113,52 @@ def streamlit_drop(
     return SingleQueryResult(cursor)
 
 
+def _default_file_callback(param_name: str):
+    from click.core import ParameterSource  # type: ignore
+
+    def _check_file_exists_if_not_default(ctx: click.Context, value):
+        if (
+            ctx.get_parameter_source(param_name) != ParameterSource.DEFAULT  # type: ignore
+            and value
+            and not Path(value).exists()
+        ):
+            raise ClickException(f"Provided file {value} does not exist")
+        return Path(value)
+
+    return _check_file_exists_if_not_default
+
+
 @app.command("deploy")
 @with_output
 @global_options_with_connection
 def streamlit_deploy(
     streamlit_name: str = typer.Argument(..., help="Name of Streamlit to deploy."),
     file: Path = typer.Option(
-        "streamlit_app.py",
+        "app.py",
```

Comment:
Please update release notes to address default value change + `--query-warehouse` requirement.

Reply:
Discussed and agreed that it's not needed in this case because we have already mentioned that `deploy` is a new command.